packages/block-library/src/column/save.js

```diff
@@ -9,13 +9,19 @@ import classnames from 'classnames';
 import { InnerBlocks } from '@wordpress/block-editor';
 
 export default function save( { attributes } ) {
-	const { verticalAlignment } = attributes;
+	const { verticalAlignment, width } = attributes;
+
 	const wrapperClasses = classnames( {
 		[ `is-vertically-aligned-${ verticalAlignment }` ]: verticalAlignment,
 	} );
 
+	let style;
+	if ( Number.isFinite( width ) ) {
+		style = { flexBasis: width + '%' };
```

Comment:
is this kses allowed (just to make sure we check as all inline style properties are not allowed)?

Reply:
> is this kses allowed (just to make sure we check as all inline style properties are not allowed)?

Good catch! It is not.

I will need to filter this set, and follow-up with a Trac ticket.

https://core.trac.wordpress.org/browser/trunk/src/wp-includes/kses.php?rev=45242#L2057

Reply:
> is this kses allowed (just to make sure we check as all inline style properties are not allowed)?

Added in 5bb641cb1881f3d7bcea6513714166c3d570a727